sdstore/sdstore.go

```diff
@@ -254,14 +254,28 @@ func (s *sdStore) Upload(u *url.URL, filePath string, toCompress bool) error {
 	}
 	err = s.putFile(encodedURL, "text/plain", zipPath)
 	if err != nil {
-		log.Printf("failed to upload file")
+		log.Printf("failed to upload file %s to store upload size = %s)", zipPath, fileSize(zipPath))
```

Comment:
```suggestion
		log.Printf("failed to upload file %s to store (upload size = %s))", zipPath, fileSize(zipPath))
```
Add parentheses for consistency